android/src/main/java/com/reactnativemembrane/MembraneWebRTCModule.kt

```diff
@@ -0,0 +1,279 @@
+package com.reactnativemembrane
+
+import expo.modules.kotlin.Promise
+import expo.modules.kotlin.functions.Coroutine
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+import expo.modules.kotlin.records.Field
+import expo.modules.kotlin.records.Record
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+class SimulcastConfig : Record {
+    @Field
+    val enabled: Boolean = false
+
+    @Field
+    val activeEncodings: List<String> = emptyList()
+}
+
+class CameraConfig : Record {
+    @Field
+    val quality: String = "VGA169"
+
+    @Field
+    val flipVideo: Boolean = false
+
+    @Field
+    val videoTrackMetadata: Map<String, Any> = emptyMap()
+
+    @Field
+    val simulcastConfig: SimulcastConfig = SimulcastConfig()
+
+    // expo-modules on Android don't support Either type
+    @Field
+    val maxBandwidthMap: Map<String, Int> = emptyMap()
+
+    @Field
+    val maxBandwidthInt: Int = 0
+
+    @Field
+    val cameraEnabled: Boolean = true
+
+    @Field
+    val captureDeviceId: String? = null
+}
+
+class MicrophoneConfig : Record {
+    @Field
+    val audioTrackMetadata: Map<String, Any> = emptyMap()
+
+    @Field
+    val microphoneEnabled: Boolean = true
+}
+
+class ScreencastOptions : Record {
+    @Field
+    val quality: String = "HD15"
+
+    @Field
+    val screencastMetadata: Map<String, Any> = emptyMap()
+
+    @Field
+    val simulcastConfig: SimulcastConfig = SimulcastConfig()
+
+    @Field
+    val maxBandwidthMap: Map<String, Int> = emptyMap()
+
+    @Field
+    val maxBandwidthInt: Int = 0
+}
+
+class MembraneWebRTCModule : Module() {
+    override fun definition() = ModuleDefinition {
+        Name("MembraneWebRTC")
+
+        Events(
```

Comment:
What about moving that bug fix from ex-29 here, since after this changes are applied I'd have to redo my PR. So wdyt about mimicing my changes here, since it is pretty short and straight forward ?